libs/category/state/src/effects/category-page.effects.ts

```diff
@@ -72,47 +73,44 @@ export class DaffCategoryPageEffects<
   @Effect()
   changeCategoryPageSize$: Observable<any> = this.actions$.pipe(
     ofType(DaffCategoryPageActionTypes.CategoryPageChangeSizeAction),
-    withLatestFrom(
+    withLatestFrom<DaffCategoryPageChangePageSize, Observable<DaffCategoryPageMetadata>>(
```

Comment:
why do you prefer casting here rather than typing the input args of the switchMap? I usually prefer typing args to casting. Seems neater

Reply:
I thought it was needed for the build but I think it was just vs code being dumb, changed